contracts/proxy-deployer/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "proxy-deployer"
+version = "0.0.0"
+authors = ["MultiversX <contact@multiversx.com>"]
+edition = "2021"
+publish = false
+
+[lib]
+path = "src/lib.rs"
+
+[dependencies.multiversx-sc]
+version = "=0.43.4"
```

Comment:
Please upgrade framework version. All the contracts have the framework version 0.44.

Reply:
Done.